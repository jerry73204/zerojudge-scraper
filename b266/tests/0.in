2 10 1
8 4
2 9
5 2
7 3
4 6
2 8
6 2
8 2
8 9
7 8
0
