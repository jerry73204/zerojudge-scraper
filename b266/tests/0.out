2 10
4 9 2 3 6 8 2 2 9 8
8 2 5 7 4 2 6 8 8 7
